code/online_challenges/src/codechef/JAIN/JAIN.cpp

```diff
@@ -0,0 +1,71 @@
+#include <bits/stdc++.h>
+
+#define fastio ios_base::sync_with_stdio(false); cin.tie(NULL);  cout.tie(NULL)
```

Comment:
Don't use these defines. Don't use a `register int`, a `std::size_t` is fine. Don't use random integer types, prefer `int` for  nearly all cases (use `long`/`long long` _only when necessary_).